tests/Metadata/Resource/Factory/FileConfigurationMetadataTest.php

```diff
@@ -153,15 +154,11 @@ public function testXmlDoesNotExistMetadataFactory()
     /**
      * @expectedException \InvalidArgumentException
```

Comment:
We found a problem :) Should be `\ApiPlatform\Exception\InvalidArgumentException` (same for all `throw new \InvalidArgumentException`).